index.js

```diff
@@ -25,7 +25,7 @@ const checkCwdOption = (options = {}) => {
 	let stat;
 	try {
 		stat = fs.statSync(options.cwd);
-	} catch (_) {
+	} catch {
```

Comment:
@sindresorhus I think this issue slipped through with the removed brackets which fails now in the latest 3.0.2 version. Should I make a PR for fixing it?